src/zherebko/index.js

```diff
@@ -24,12 +24,25 @@ export default function() {
     let maxIndex = 0;
     dag.eachLinks(({ data }) => {
       minIndex = Math.min(minIndex, data.index);
-      maxIndex = Math.max(maxIndex, data.index);
+      maxIndex = Math.max(maxIndex, data.index > 0 ? data.index : 1);
     });
     let maxLayer = ordered.length - 1;
+
     dag.each((node) => {
-      node.x = (-minIndex / (maxIndex - minIndex)) * width;
-      node.y = (node.layer / maxLayer) * height;
+      if (maxLayer === 0) {
+        // If there's just 1 node
+        // we should just center it
+        node.x = width / 2;
+        node.y = height / 2;
```

Comment:
This is an important edge case, but we can escape much later. If there's only one node, we know for a fact what it's coordinate is without doing anything. Immediately after `ordered` is initialized, we should just check if it's length 1, and if so, set x and y to this and return the dag.